testing/src/test/java/org/oppia/android/testing/robolectric/OppiaShadowTrafficStatsTest.kt

```diff
@@ -0,0 +1,104 @@
+package org.oppia.android.testing.robolectric
+
+import android.app.Application
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Singleton
+
+private val oppiaShadowTrafficStats by lazy { OppiaShadowTrafficStats() }
+
+/** Tests for [OppiaShadowTrafficStats]. */
+// FunctionName: test names are conventionally named with underscores.
+@Suppress("FunctionName")
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  manifest = Config.NONE,
+  sdk = [Build.VERSION_CODES.P],
+  shadows = [OppiaShadowTrafficStats::class]
+)
+class OppiaShadowTrafficStatsTest {
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @After
+  fun tearDown() {
+    // Make sure this is reset between tests.
+    OppiaShadowTrafficStats.reset()
+  }
+
+  @Test
+  fun testCustomShadow_initialState_returnsDefaultValuesForTxAndRxBytes() {
+    assertThat(OppiaShadowTrafficStats.getUidRxBytes(0)).isEqualTo(0L)
+    assertThat(OppiaShadowTrafficStats.getUidTxBytes(0)).isEqualTo(0L)
+
+    OppiaShadowTrafficStats.reset()
```

Comment:
Is this needed if it's also being called in tearDown?

Reply:
removed.